db/seeds.rb

```diff
@@ -5,3 +5,16 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
```

Comment:
Ciekawe czy można to usunąć  🧐

Reply:
Nie wiem trzeba się zapytać ❓ 

@jerryskye ❓

Reply:
Można :ok_hand: